src/cephadm/cephadm

```diff
@@ -95,7 +95,7 @@ class TimeoutExpired(Error):
 
 class Ceph(object):
     daemons = ('mon', 'mgr', 'mds', 'osd', 'rgw', 'rbd-mirror',
-               'crash')
+               'crash', 'iscsi')
```

Comment:
Hm. maybe we shoudn't add `'iscsi'` to the Ceph daemons and instead have it as a separate thing?

Reply:
OK, I'll take a look. Only problem is ceph-iscsi is apart of ceph (well at least the project namespace) and we still need a ceph.conf, keyring etc, so much of the same code is required. But I guess I duplicate all of that. I know it isn't nice, I guess its not so great either way from a maintenance perspective.